app/src/androidTest/java/com/google/maps/android/compose/MapInColumnTests.kt

```diff
@@ -116,9 +116,14 @@ class MapInColumnTests {
     fun testScrollColumn_MapCameraRemainsSame() {
         initMap()
         // Check that the column scrolls to the last item
-        composeTestRule.onRoot().performTouchInput { swipeUp() }
+        composeTestRule.onRoot().performTouchInput {
+            swipeUp(
+                startY = (bottom - top) / 2
+            )
+        }
+
         composeTestRule.waitForIdle()
-        composeTestRule.onNodeWithTag("Item 1").assertIsNotDisplayed()
+        // composeTestRule.onNodeWithTag("Item 1").assertIsNotDisplayed()
```

Comment:
AFAIK this specific test started failing on CI starting with https://github.com/googlemaps/android-maps-compose/pull/171, so I'm guessing something there triggered it with the change in `compileSdk`, `targetSdk`, or compose dependencies.

Looks like the CI android-emulator-runner in instrumentation-test.yml has a newer version than `uses: reactivecircus/android-emulator-runner@v2`, v2.25.0 - maybe try that?

https://github.com/ReactiveCircus/android-emulator-runner/releases/tag/v2.25.0

We could also try increasing `api-level: 29` to `30`, `31` or `32`.

Looking more, it might be worth mirroring the accompanist setup:
https://github.com/google/accompanist/blob/main/.github/workflows/device-tests.yml

They have a matrix set up to test APIs 22-29. I'm curious if this would pass on other emulator versions.

Reply:
FYI, I tried changing to `disable-animations: false` but got the same test failure.

Reply:
I was doing some more testing and stumbled on some output earlier in the build in [this run](https://github.com/googlemaps/android-maps-compose/runs/7365405448?check_suite_focus=true):

>WARNING:We recommend using a newer Android Gradle plugin to use compileSdk = 32
This Android Gradle plugin (7.0.4) was tested up to compileSdk = 31
This warning can be suppressed by adding
    android.suppressUnsupportedCompileSdk=32
to this project's gradle.properties

I updated Gradle and AGP in PR https://github.com/googlemaps/android-maps-compose/pull/181, so even though that PR didn't fix the flaky test out of the gate I think it's better to merge that PR, rebase this PR on main, and then test further so we're working off of the latest AGP with `compileSdk 32`.

Reply:
This passes tests now. We figured out that the test flakiness was due to the way we were loading the API key secret.